### DISCUSSION TITLE
What does mean the time closest to the branch name?

### DISCUSSION THREAD

**Discussion by estebancastrosola** [jesseduffield/lazygit#2209] [Q&A]
![image](https://user-images.githubusercontent.com/30086507/194567465-e3e98a2c-1b87-4e2d-bc97-2fe2d8c0b1d0.png)

In local branch there is a number with a letter and this changes when i checkout between diferent branches but no idea what represents.

Thanks you.

**Comment by ooloth** [comment]
The number and letter are a short way of expressing how long its been since that branch was updated:

`1w` = "one week"
`4d` = "four days"
`2y` = "two years" 

etc.
